app/views/candidate_mailer/find_has_opened.erb

```diff
@@ -0,0 +1,20 @@
+<% if @application_form.first_name.present? %>
+  Dear <%= @application_form.first_name %>,
+<% end %>
+
+You can now find teacher training courses starting in the <%= @academic_year %> academic year.
+
+Find your course and get your application ready:
+
+https://www.gov.uk/find-postgraduate-teacher-training-courses
+
+You can submit from 9am on <%= @apply_opens %>.
```

Comment:
Is the 9am time specified in code anywhere?

Reply:
It is yea. That's the actual time that everything will start working for the new cycle

Reply:
Oh I meant should we insert the time based on a constant defined elsewhere rather than hard-code it. But it's fine... 😄